ffv1.md

```diff
@@ -23,17 +23,21 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 ## Definitions
 
 -------- --------------------------------------------------------------
+`Sample`: The smallest addressable representation of a primary color in a frame. Examples of sample are Luminance, Blue Chrominance, Red Chrominance, Alpha, Red, Green, Blue.
```

Comment:
I think the term `primary color` is misleading here, since a `Y` sample does not represent color. I see some language from [RFC4175](https://tools.ietf.org/search/rfc4175) which discusses this like:
> A sample may represent a color component or a luminance component of the video.